configuration/janusgraph-env.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is this file derived from an Apache Ambari file? I'm not clear on why this is licensed by ASF rather than JanusGraph. Same consideration for many other files in this PR.

Reply:
I will update all the licensing. Originally the goal was to add this through ambari so I used the existing plugins as a reference.